config/control_with_ur5_arm.yaml

```diff
@@ -0,0 +1,68 @@
+husky_joint_publisher:
+  type: "joint_state_controller/JointStateController"
```

Comment:
unpleasant to repeat all the stuff from control.yaml. Can you make a separate control_ur5.yaml?